src/main/java/com/minecolonies/coremod/client/gui/map/WindowColonyMap.java

```diff
@@ -0,0 +1,337 @@
+            dragView.addChild(colonyPane);
+            coloniesImages.put(colonyInfo, colonyPane);
+            PaneBuilders.tooltipBuilder().hoverPane(colonyPane)
+              .append(new StringTextComponent("Owner:" + colonyInfo.getOwner()))
+              .appendNL(new StringTextComponent("Coordinates: " + colonyInfo.getCenter().getX() + "X, " + colonyInfo.getCenter().getZ() + "Z"))
+              .appendNL(new StringTextComponent("Citizens: " + colonyInfo.getCitizencount()))
+              .build();
+        }
+
+        if (currentScale < COLONY_DETAIL_SCALE)
+        {
+            colonyPane.setPosition(worldPosToUIPos(colonyInfo.getCenter()).getX() - colonyPane.getWidth() / 2,
+              worldPosToUIPos(colonyInfo.getCenter()).getZ() - colonyPane.getHeight() / 2);
+            colonyPane.on();
+        }
+        else
+        {
+            colonyPane.off();
+        }
+    }
+
+    /**
+     * Adds the central blob identifying the current position
+     */
+    private void addCenterPos()
+    {
+        final Image citizenImage = new Image();
+        citizenImage.setImage(new ResourceLocation(Constants.MOD_ID, "textures/gui/red_wax_actions.png"));
+        citizenImage.setSize(16, 16);
+        citizenImage.setPosition(worldPosToUIPos(playerPos).getX(), worldPosToUIPos(playerPos).getZ());
+        dragView.addChild(citizenImage);
+    }
+
+    /**
+     * Generate a list of icons for the buildings
+     *
+     * @param buildingView
+     * @return
+     */
+    private void updateBuildingView(final IBuildingView buildingView)
+    {
+        ItemIcon uiBuilding = buildings.get(buildingView);
+        if (uiBuilding == null)
+        {
+            uiBuilding = new ItemIcon();
+
+            uiBuilding.setID(buildingView.getID().toShortString());
+            uiBuilding.setSize(11 + buildingView.getBuildingLevel(), 11 + buildingView.getBuildingLevel());
+            final ItemStack item = buildingView.getBuildingType().getBuildingBlock().asItem().getDefaultInstance();
+            uiBuilding.setItem(item);
+            dragView.addChild(uiBuilding);
+
+            AbstractTextBuilder.TooltipBuilder tooltip = PaneBuilders.tooltipBuilder();
+            tooltip.hoverPane(uiBuilding)
+              .append(window.getScreen().getTooltipFromItem(item).get(0)).append(new StringTextComponent(" : " + buildingView.getBuildingLevel()))
+              .appendNL(new StringTextComponent("Coordinates: " + buildingView.getID().getX() + "X, " + buildingView.getID().getZ() + "Z"))
+              .appendNL(new StringTextComponent("Citizens: " + buildingView.getAllAssignedCitizens().size()));
+
+            for (int id : buildingView.getAllAssignedCitizens())
+            {
+                final ICitizenDataView dataView = building.getColony().getCitizen(id);
+                if (dataView != null)
+                {
+                    tooltip.appendNL(new StringTextComponent(dataView.getName()));
```

Comment:
Might want to tab that using eg. two spaces? (not sure if it's trimmed or not)
```suggestion
                    tooltip.appendNL(new StringTextComponent("  " + dataView.getName()));
```